src/Integration.Vsix/SonarLintTagger/IssueTagger.cs

```diff
@@ -218,15 +241,16 @@ private void UpdateMarkers(ITextSnapshot currentSnapshot, IssuesSnapshot snapsho
 
         public IEnumerable<ITagSpan<IErrorTag>> GetTags(NormalizedSnapshotSpanCollection spans)
         {
+            Debug.Assert(!isDisposed, "Not expecting GetTags to be called after the tagger has been disposed");
+
             if (snapshot == null)
             {
                 return Enumerable.Empty<ITagSpan<IErrorTag>>();
             }
 
             return snapshot.IssueMarkers
-                .Select(issue => issue.Span)
-                .Where(spans.IntersectsWith)
-                .Select(span => new TagSpan<IErrorTag>(span, new ErrorTag(PredefinedErrorTypeNames.Warning)));
+                .Where(marker => spans.IntersectsWith(marker.Span))
+                .Select(marker => new TagSpan<IErrorTag>(marker.Span, new ErrorTag(PredefinedErrorTypeNames.Warning, marker.Issue.Message)));
```

Comment:
There is a built-in _QuickInfoSource_ implementation that will automatically include tooltip information from _IErrorTags_. Here, we're using a different override of the _ErrorTag_ constructor to pass in the issue message as the tooltip info.